src/frontend/components/UI/Sidebar/index.css

```diff
@@ -60,6 +60,14 @@
 
 .Sidebar__item:hover {
   background-color: var(--navbar-active-background);
+  transition: 0.2s;
+  scale: 1.02;
+  width: 99%;
```

Comment:
If this isn't set, the hovered item will expand the width of the sidebar due to which a horizontal scrollbar will appear below the Heroic version text.

Reply:
We can use `overflow-x: hidden` in the `.Sidebar` selector to avoid the need of these `width`s. It would be more maintainable.

Reply:
Alright,  I'll look into this.

Reply:
I've used `overflow-x: hidden` now.